activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -366,6 +366,16 @@ def test_create_with_reset
     assert_equal "Jamis", jamis.name
   end
 
+  def test_create_with_nested_attributes
+    assert_difference("Project.count", 1) do
```

Comment:
Isolation test would be failed without `require "models/project"`.

Reply:
oh, my bad. I thought it was included.